opensrp-giz-malawi/src/main/assets/ec_client_fields.json

```diff
@@ -45,6 +45,27 @@
             "field": "identifiers.zeir_id"
           }
         },
+        {
+          "column_name": "opensrp_id",
+          "type": "Client",
+          "json_mapping": {
+            "field": "identifiers.M_ZEIR_ID"
+          }
+        },
+        {
+          "column_name": "register_id",
+          "type": "Client",
+          "json_mapping": {
+            "field": "identifiers.M_ZEIR_ID"
+          }
+        },
+        {
+          "column_name": "zeir_id",
+          "type": "Client",
+          "json_mapping": {
+            "field": "identifiers.M_ZEIR_ID"
+          }
+        },
```

Comment:
@roywanyaga Is their a reason we have 3 columns mapping to the same identifier?

Reply:
@dubdabasoduba These three columns are being assigned the same value for the child client. My assumption was that all the three columns could be needed for the mother/guardian als:

**in db**
<img width="577" alt="image" src="https://user-images.githubusercontent.com/84201465/149768260-0a316174-31f4-467e-8fc7-f316eebe3c03.png">

**in ec_client** 
 {
          "column_name": "register_id",
          "type": "Client",
          "json_mapping": {
            "field": "identifiers.zeir_id"
          }
        },
        {
          "column_name": "opensrp_id",
          "type": "Client",
          "json_mapping": {
            "field": "identifiers.M_ZEIR_ID"
          }
        },
        ......
        
          {
          "column_name": "zeir_id",
          "type": "Client",
          "json_mapping": {
            "field": "identifiers.zeir_id"
          }
        },